src/GoalioRememberMe/Authentication/Adapter/Cookie.php

```diff
@@ -77,6 +77,16 @@ public function authenticate(AuthEvent $e)
 
         $userObject = $this->getUserMapper()->findById($cookie[0]);
 
+        if(!$userObject) {
+            $this->getRememberMeMapper()->removeAll($cookie[0]);
+            $this->getRememberMeService()->removeCookie();
+            $this->setSatisfied(false);
+
+            $e->setCode(AuthenticationResult::FAILURE)
+            ->setMessages(array('User not found.'));
```

Comment:
It's not in constants in the other cases, so it's not in the scope of this ticket to move it :)